adr/012-configs.md

```diff
@@ -0,0 +1,37 @@
+# 12. Partner configuration
+
+Date: 2024-12-12
+
+## Decision
+
+We will store partner config settings in an Azure container
```

Comment:
nit: consider being more specific here, .i.e. "We will store partner sftp config settings..."

Reply:
Fixed, thanks!